src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -343,7 +343,7 @@ def add_host(self, host_spec: HostSpec) -> OrchResult[str]:
         """
         raise NotImplementedError()
 
-    def remove_host(self, host: str) -> OrchResult[str]:
+    def remove_host(self, host: str, force: bool, offline: bool) -> OrchResult[str]:
```

Comment:
might want to update `remove_host()` defined in `src/pybind/mgr/test_orchestrator/module.py` as well. i have
```
2021-07-30T05:27:21.591 DEBUG:teuthology.orchestra.run.smithi185:> sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 120 ceph --cluster ceph orch host rm hostname
2021-07-30T05:27:21.876 INFO:tasks.ceph.mgr.x.smithi185.stderr:2021-07-30T05:27:21.875+0000 7fb6c6e4c700 -1 mgr handle_command module 'orchestrator' command handler threw exception: remove_host() takes 2 positional arguments but 4 were given
2021-07-30T05:27:21.876 INFO:tasks.ceph.mgr.x.smithi185.stderr:2021-07-30T05:27:21.875+0000 7fb6c6e4c700 -1 mgr.server reply reply (22) Invalid argument Traceback (most recent call last):
2021-07-30T05:27:21.877 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/mgr_module.py", line 1563, in _handle_command
2021-07-30T05:27:21.877 INFO:tasks.ceph.mgr.x.smithi185.stderr:    return self.handle_command(inbuf, cmd)
2021-07-30T05:27:21.877 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 167, in handle_command
2021-07-30T05:27:21.877 INFO:tasks.ceph.mgr.x.smithi185.stderr:    return dispatch[cmd['prefix']].call(self, cmd, inbuf)
2021-07-30T05:27:21.878 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/mgr_module.py", line 414, in call
2021-07-30T05:27:21.878 INFO:tasks.ceph.mgr.x.smithi185.stderr:    return self.func(mgr, **kwargs)
2021-07-30T05:27:21.878 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 107, in <lambda>
2021-07-30T05:27:21.880 INFO:tasks.ceph.mgr.x.smithi185.stderr:    wrapper_copy = lambda *l_args, **l_kwargs: wrapper(*l_args, **l_kwargs)  # noqa: E731
2021-07-30T05:27:21.880 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 96, in wrapper
2021-07-30T05:27:21.880 INFO:tasks.ceph.mgr.x.smithi185.stderr:    return func(*args, **kwargs)
2021-07-30T05:27:21.881 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/orchestrator/module.py", line 352, in _remove_host
2021-07-30T05:27:21.881 INFO:tasks.ceph.mgr.x.smithi185.stderr:    raise_if_exception(completion)
2021-07-30T05:27:21.881 INFO:tasks.ceph.mgr.x.smithi185.stderr:  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 224, in raise_if_exception
2021-07-30T05:27:21.882 INFO:tasks.ceph.mgr.x.smithi185.stderr:    raise e
2021-07-30T05:27:21.882 INFO:tasks.ceph.mgr.x.smithi185.stderr:TypeError: remove_host() takes 2 positional arguments but 4 were given
2021-07-30T05:27:21.882 INFO:tasks.ceph.mgr.x.smithi185.stderr:
2021-07-30T05:27:21.891 DEBUG:teuthology.orchestra.run:got remote process result: 22
```

in https://pulpito.ceph.com/kchai-2021-07-30_04:58:53-rados-wip-kefu-testing-2021-07-30-0947-distro-basic-smithi/6302956/